config/nav.yml

```diff
@@ -129,6 +129,9 @@ nav:
       # Serving - troubleshooting
       - Troubleshooting:
         - Debugging application issues: serving/troubleshooting/debugging-application-issues.md
+    # Eventing reference docs
```

Comment:
I think the comment here is wrong, but it may not be necessary / structural

Reply:
Good catch, thank you 🙂